templates/docker-compose.yml

```diff
@@ -1,54 +1,100 @@
-version: '2'
+version: "3.3"
```

Comment:
I changed this around to more closely match our dockerfile in lemmy.

Reply:
FYI its 3.7 now, but apparently the version doesnt matter anyway.

https://github.com/LemmyNet/lemmy/pull/3248